index.html

```diff
@@ -29,34 +29,168 @@ <h2 class="title ocupation">
         </p>
         <div class="social-container">
           <ul>
-            <li><a href="#"><img src="/images/twitter.png" alt=""></a></li>
-            <li><a href="#"><img src="/images/Linkedin-icon.png" alt=""></a></li>
-            <li><a href="#"><img src="/images/medium.png" alt=""></a></li>
-            <li><a href="#"><img src="/images/github.png" alt=""></a></li>
-            <li><a href="#"><img src="/images/angellist.png" alt=""></a></li>
+            <li>
+              <a href="#"><img src="/images/twitter.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="/images/Linkedin-icon.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="/images/medium.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="/images/github.png" alt=""></a>
+            </li>
+            <li>
+              <a href="#"><img src="/images/angellist.png" alt=""></a>
+            </li>
           </ul>
           <a class="down-icon" href="#">
             <img src="/images/down.png" alt="down-icon">
           </a>
         </div>
       </section>
+      <section class="works">
+        <h2 class="section-title">
+          My Recent Works
+        </h2>
+        <div class="card-container">
+          <div class="card">
+            <div class="card-img">
+              <p>img</p>
```

Comment:
- Please remove the `<p>img</p>` as it shows on the image inserted for the project, which is not required in this project.
- PLease the text `see project` and `Get My Resume` should have a color of  `#fff` as it is in the Figma design, currently yours is having a color of black. please fix it to use the appropriate colors.
- PLease for the `Multipost story...` the color should be `color: #3A4A42;` as it is in the figma design. 

##### Figma design 
![Screenshot from 2022-07-02 16-05-49](https://user-images.githubusercontent.com/58771507/177006140-1384c243-804d-4dd6-96b3-0699bcae443b.png)

##### Your design 
![Screenshot from 2022-07-02 16-09-23](https://user-images.githubusercontent.com/58771507/177006156-17dbf4db-c01d-4fa0-96bf-7344d06ffcd7.png)